draft-ietf-rats-corim.md

```diff
@@ -1066,6 +1066,182 @@ The following describes each member of the `concise-bom-tag` map.
   The `$$concise-bom-tag-extension` extension socket is empty in this
   specification.
 
+# CoRIM-based Evidence Verification
+
+The verification procedure is divided into three separate phases:
+
+* Appraisal Context initialisation
+* Evidence collection
+* Evidence appraisal
+
+At a few well-defined points in the procedure, the Verifier behaviour will depend on the specific CoRIM profile.
+Each CoRIM profile SHALL fully describe the expected Verifier behaviour for each of those well-defined points.
+
+Note that what follows is not meant to describe a real implementation.
+In particular, it is expected that the cost associated with the initialisation phase can be amortised across multiple appraisals.
+Implementers are free to do as they please, as long as the stated invariants are not broken.
+
+In the following, if a MUST-level requirement is violated, the entire procedure is aborted.
```

Comment:
this is always true. Could have been removed, if this would be normative text. But is an example. Doing both is possible (actually...) but confusing. Normative sections have to be separated from informative sections.

Reply:
> this is always true. Could have been removed, if this would be normative text. But is an example. Doing both is possible (actually...) but confusing. Normative sections have to be separated from informative sections.

We could debate on whether it's a necessity -- there could be more complex control flow decisions based on a requirement violation than just "bail out!".
But more generally, it doesn't hurt to make an explicit statement here, at the very beginning of a long and complex set of actions.

Reply:
Actually, I want to correct myself.  I think we should drop L1089 altogether not because it is redundant, but because it is misleading: for example, when later on we say _"corims outside their validity period MUST be discarded"_ we don't want to kill the appraisal if a corim is found to be past its expiration, we just want to avoid pulling it in...